File: atcoder.jp/contests/mujin-pc-2016/tasks/mujin_pc_2016_a/main.go
package main

import (
	"fmt"
	"strings"
)

func YesNo(b bool) string {
	if b {
		return "Right"
	} else {
		return "Left"
	}
}

func main() {
	var c string
	fmt.Scanf("%s", &c)

	ans := strings.Contains("OPKL", c)

	fmt.Println(YesNo(ans))
}

File: atcoder.jp/contests/mujin-pc-2016/tasks/mujin_pc_2016_a/main.cpp
#include <bits/stdc++.h>
using namespace std;

bool contains(const string &search_string, const string &s) {
  for (size_t i = 0; i < s.size(); i++)
    if (s.substr(i, search_string.size()) == search_string) return true;
  return false;
}

int main() {
  string c;
  cin >> c;

  bool ans = contains(c, "OPKL");

  cout << (ans ? "Right" : "Left") << endl;
}
